_data/nav.yml

```diff
@@ -36,6 +36,7 @@ navigation:
     - Dofus
     - Dragon Nest
     - Dungeons and Dragons Online
+	- Elder Scrolls Online International
```

Comment:
This file uses spaces. Additionally this category should just be 'Elder Scrolls Online'